pkg/workloads/console/integration/integration_test.go

```diff
@@ -102,16 +105,56 @@ var _ = Describe("Console", func() {
 			}
 			By("Reconcile done")
 		}
-
-		waitForSuccessfulReconcile(2)
 	})
 
 	AfterEach(func() {
 		cancel()
 		teardown()
 	})
 
+	Describe("enforcing valid timeout values", func() {
```

Comment:
We could potentially add another test case that ensures that if a console is created with a timeout that's non-zero and lower than the max, then it isn't altered.

Could be overkill. WDYT?

Reply:
Sure, happy to add that.